### ISSUE TITLE
Suggestion: have EasyOffset for quest 2 natively 

### ISSUE THREAD

**Issue by shockingfargroundx** [Reezonate/EasyOffset#13]
Would be pretty cool

**Comment by shockingfargroundx** [comment]
Please

**Comment by polaire801** [comment]
+1

I'm interested as well.

**Comment by saiko361** [comment]
Would be awesome, I want to switch from Steam to Quest 3, missing EasyOffset is the only thing holding me back :D

**Comment by waffshappen** [comment]
@Reezonate is this still under Consideration? Not having EasyOffset makes Beat Saber on the go almost impossible, and the basic both-hand offsets in native beat saber are not remotely good enough. 

I'd be willing to put 100€ towards that original ko-fi bounty (but towards a Quest 3 Version instead) if you still have a way to donate that works.

**Comment by Reezonate** [comment]
@waffshappen yes, it is planned. Right now I'm working on a full PC mod rewrite, which includes all the necessary changes to make mod easier to port to C++ (quest mods are written in C++)

As for donations, you can use [DonationAlerts](https://www.donationalerts.com/r/reezonate) (please let me know it is also broken like ko-fi)
